### ISSUE TITLE
vimproc_linux64.so is not found

### ISSUE THREAD

**Issue by in-in** [wsdjeg/SpaceVim#544]
## Expected behavior, english is recommend
No errors

## Environment Information
- OS: Linux Mint 18.1 Cinnamon 64-bit
- neovim version: 0.1.7

## The reproduce ways from Vim starting (Required!)
run nvim

## Output of the ':SPDebugInfo!'
clipboard: provider is not available

please post log below, if you want me reproduce your issue quickly, post your custom config here will be better.

## Screenshots
![7b26f](https://cloud.githubusercontent.com/assets/8797432/26218991/c289cfd6-3c15-11e7-9df9-a1f7d0c7701c.png)

If you have any screenshots for this issue please upload here. BTW you can use https://asciinema.org/ for recording video  in terminal.

**Comment by wsdjeg** [comment]
you need compile vimproc. please read the doc of vimproc

**Comment by in-in** [comment]
I'm read it.
This `:call dein#add('Shougo/vimproc.vim', {'build' : 'make'})` don't work. Nothing happens

**Comment by Shougo** [comment]
I think your compiler configuration is broken.
You should execute `make` in vimproc directory.

I have read the same problem in SpaceVim gitter room.

```
src/proc.c:10:19: fatal error: stdio.h: No such file or directory
let me find article on setup for building from source... brb...
oh
ok, seems to have worked!
thanks Shougo!
Installing all build from source packages g++ etc, will take a while. Thanks for helping.
```

**Comment by Shougo** [comment]
Note: He also uses Mint.

**Comment by in-in** [comment]
I installed `build-essential` before SpaceVim and problem were gone

**Comment by ZihengJiang** [comment]
Same issue again on OSX 10.15 Catalina. I think it is because this problem: https://stackoverflow.com/questions/58278260/cant-compile-a-c-program-on-a-mac-after-upgrading-to-catalina-10-15

**Comment by Habib-Rezaie** [comment]
> happens

I do have this problem also, running make makes nothing happen!

**Comment by Shougo** [comment]
Please execute make command on the directory manually.

**Comment by scaamanho** [comment]
Same issue on ubuntu WSL instaling neovim,

just go directory :~/.SpaceVim/bundle/vimproc.vim/ and type `make` to create it.

References:
<https://github.com/Shougo/vimproc.vim>

**Comment by wsdjeg** [comment]
@scaamanho Thanks for your tips.

**Comment by vverman** [comment]
Thanks for the valuable tip @scaamanho!

**Comment by shashinvision** [comment]
> Same issue on ubuntu WSL instaling neovim,
> 
> just go directory :~/.SpaceVim/bundle/vimproc.vim/ and type `make` to create it.
> 
> References:
> https://github.com/Shougo/vimproc.vim

thank you, my problem is resolved with your help in Kubuntu 20.04

**Comment by andersonbosa** [comment]
thanks 2all, solved w your tip too @scaamanho 
me: `n/a ManjaroLinux "Manjaro Linux" 21.1.0 Pahvo`
